.jscsrc

```diff
@@ -1,48 +1,4 @@
 {
-  "disallowKeywords": ["with"],
-  "disallowKeywordsOnNewLine": ["else"],
-  "disallowMixedSpacesAndTabs": true,
-  "disallowMultipleVarDecl": "exceptUndefined",
-  "disallowNewlineBeforeBlockStatements": true,
-  "disallowQuotedKeysInObjects": true,
-  "disallowSpaceAfterObjectKeys": true,
-  "disallowSpaceAfterPrefixUnaryOperators": true,
-  "disallowSpacesInFunction": {
-      "beforeOpeningRoundBrace": true
-  },
-  "disallowSpacesInsideParentheses": true,
-  "disallowTrailingWhitespace": true,
-  "maximumLineLength": null,
-  "requireCamelCaseOrUpperCaseIdentifiers": true,
-  "requireCapitalizedComments": true,
-  "requireCapitalizedConstructors": true,
-  "requireCurlyBraces": true,
-  "requireSpaceAfterKeywords": [
-      "if",
-      "else",
-      "for",
-      "while",
-      "do",
-      "switch",
-      "case",
-      "return",
-      "try",
-      "catch",
-      "typeof"
-  ],
-  "requireSpaceAfterLineComment": true,
-  "requireSpaceAfterBinaryOperators": true,
-  "requireSpaceBeforeBinaryOperators": true,
-  "requireSpaceBeforeBlockStatements": true,
-  "requireSpaceBeforeObjectValues": true,
-  "requireSpacesInFunction": {
-      "beforeOpeningCurlyBrace": true
-  },
-  "requireTrailingComma": {
-      "ignoreSingleLine": true
-  },
-  "requireBlocksOnNewline": true,
-  "validateIndentation": 2,
-  "validateLineBreaks": "LF",
-  "validateQuoteMarks": "'"
+  "preset": "node-style-guide",
+  "maximumLineLength": null
```

Comment:
I'm somewhat indifferent. I usually find a lot of overlap between JSCS and ESLint, which can lead to confusion if you dare specify different indenting rules in multiple places. So I guess my only concern here would be if we use a **node-style-guide** JSCS preset, but then change our ESLint rules to use XO (which may have some conflicts).

Although we can always extend the **node-style-guide** JSCS preset, override the conflicting rules after migrating to XO, and then republishing our own custom **jscs-preset-gulp** module once everything is green and passing tests.

Or, if you want to dump JSCS, we can use something like [polyjuice](https://www.npmjs.com/package/polyjuice) to convert the .jscs config file into .eslintrc to see where the overlapping rules are, and then see if "eslint:recommended" or "xo" already define those rules (or conflicting rules).